zap/src/main/java/org/zaproxy/zap/extension/script/OptionsScriptPanel.java

```diff
@@ -62,6 +65,7 @@ private void initialize() {
 
         this.add(new ZapHtmlLabel(Constant.messages.getString("options.script.label.dirs")), gbc);
 
+        this.add(enableLoadedScripts, gbc);
```

Comment:
Could you add a screenshot?

Reply:
![2022-11-29_18-27](https://user-images.githubusercontent.com/103114290/204555640-a5fa6749-d16c-41fd-9260-c957258238ec.png)

Reply:
Thanks